### DISCUSSION TITLE
Expose Collector via fully qualified domain name (FQDN) & TLS

### DISCUSSION THREAD

**Discussion by naveenkumarthangaraj** [open-telemetry/opentelemetry-collector#5540] [Q&A]
HI Team,

We are in process of the setting up the centralized collector for application push the traces. We had tested it with IP Address & TLS and it worked fine without any issues.

Now, we are planning to move these changes to Testing then to production. As per standard of our Organization, Security and Network team wants to us FQDN and TLS 1.2 with 443 or 8443 as port exposed outside our K8S Clusters. 

Can you please guide with documentation or steps to setup in clusters? 

We had reviewed the below documentation from OpeTel but it not help. can you please help here?

https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md 

Above steps for the mTLS but does it expose with QDN and TLS 1.2?

Thanks,
Naveen T

**Comment by jpkrohling** [comment]
Answered at #5545
